.devcontainer/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.8
+
+* [added extension for VS Code eslinter](https://github.com/gofractally/psibase-contributor/pull/36/commits/0004449ceef6a560eb5cd24969ae1647d0ebf7e5)

```

Comment:
But is it an eslinter extension?
![image](https://github.com/user-attachments/assets/c703903a-2434-449c-9cba-dc733f458790)